### ISSUE TITLE
Allways asked for keyfile

### ISSUE THREAD

**Issue by vmaxdragon** [sbidy/KeePass-KeyManager#8]
Hi, I'm testing this plugin for using a smart card.
I generated a new database as described in the tutorial.
But if I open the database allways I have to chose the keyfile.
Is this normal?

Best regards,
Harry

**Comment by sbidy** [comment]
Please rename the p7mkey-File to the same name as the database file name and save the (encrypted) key file in the same folder.

Example: C:\Test.kdbx (database) - C:\Test.p7mkey (key file).

**Comment by vmaxdragon** [comment]
Thanks. This works. But is this not unsecure?
I thought, the keyfile must not be in the same folder as the database...

**Comment by sbidy** [comment]
Nope, the Key-File is encrypted (enveloped) with your (certificate) Pub.-Key. A recommendation is, to protect the Priv.-Key with a PIN or password.
For the best security you should use a SmartCard/HSM for protecting the private key.
